imap_processing/lo/l1b/lo_l1b.py

```diff
@@ -158,6 +163,54 @@ def get_avg_spin_durations(
     return avg_spin_durations
 
 
+def get_spin_phase(l1a_de: xr.Dataset) -> Union[np.ndarray[np.float64], Any]:
```

Comment:
I think we've been using "phase" to be the `[0, 1)` value, and "angle" to be the `[0, 360)` version. So do we want to call this function `get_spin_angle()` instead?